src/jni_core/src/lib.rs

```diff
@@ -1024,6 +1024,9 @@ extern "system" fn Java_com_risingwave_java_binding_Binding_sendCdcSourceErrorTo
     msg: JString<'a>,
 ) -> jboolean {
     execute_and_catch(env, move |env| {
+        if msg.is_null() {
```

Comment:
How about doing this when the following `get_string` return with err? We can also print a warning log there.

Reply:
refined.